common/persistence/src/main/resources/db/migration/postgres/V1__createTables.sql

```diff
@@ -1,6 +1,5 @@
 CREATE TABLE diagnosis_key (
-    id bigserial PRIMARY KEY,
-    key_data bytea,
+    key_data bytea PRIMARY KEY,
```

Comment:
This needs to be an own V3 migration, else this will crash on existing systems. (Use `ALTER TABLE`)